### PR TITLE
Extended `upgrade_nrml `to upgrade the source model to NRML 0.5

### PR DESCRIPTION
**Pull request by micheles**
Needed for https://github.com/gem/oq-engine/issues/2099.

### CODE REVIEW

File: openquake/commonlib/sourceconverter.py [hunk 1]
```diff
@@ -55,6 +56,108 @@
...
+            if not sg.sources:
+                # we append just one source per SourceGroup, so that
+                # the memory occupation is insignificant
+                sg.sources.append(src)
+
+        # return SourceGroups, ordered by TRT string
+        return sorted(source_stats_dict.values())
+
+    def __init__(self, trt, sources=None,
+                 min_mag=None, max_mag=None, id=0, eff_ruptures=-1):
```
Comment (ptormene) [inline openquake/commonlib/sourceconverter.py]: This is not consistent with the above docstring

File: openquake/calculators/ucerf_event_based.py [hunk 2]
```diff
@@ -637,12 +640,12 @@ def convert_UCERFSource(self, node):
 
 
 @parallel.litetask
-def compute_ruptures(branch_info, source, sitecol, oqparam, monitor):
+def compute_ruptures(branch_info, ucerf, sitecol, oqparam, monitor):
```
Comment (ptormene) [inline openquake/calculators/ucerf_event_based.py]: Please fix the docstring

**Comment by ptormene** [comment]
Apart from a couple of docstrings, LGTM

File: openquake/commonlib/sourceconverter.py (hunk 1 above)
Comment (micheles) [inline openquake/commonlib/sourceconverter.py]: Fixed

File: openquake/calculators/ucerf_event_based.py (hunk 2 above)
Comment (micheles) [inline openquake/calculators/ucerf_event_based.py]: Done

**Pull request merged**